### PR TITLE
Adapt to specberus expectations for joint publications

### PR DESCRIPTION
**Pull request by dontcallmedom**
cf https://github.com/w3c/specberus/blob/master/lib/rules/sotd/pp.js#L23

### CODE REVIEW

**Comment by halindrome** [comment]
LGTM

File: js/w3c/headers.js [hunk 1]
```diff
@@ -452,9 +452,9 @@ define(
                     });
                     var pats = [];
                     for (var i = 0, n = conf.wg.length; i < n; i++) {
-                        pats.push("<a href='" + conf.wgPatentURI[i] + "' rel='disclosure'>" + conf.wg[i] + "</a>");
+                        pats.push("a <a href='" + conf.wgPatentURI[i] + "' rel='disclosure'>" + "public list of any patent disclosures  (" + conf.wg[i] + ")</a>");
```
Comment (marcoscaceres) [inline js/w3c/headers.js]: line length, please :)

**Comment by marcoscaceres** [comment]
Apart from the nit, LGTM

**Comment by marcoscaceres** [comment]
Released in 3.2.66

**Pull request merged**